app/main/views/services.py

```diff
@@ -180,25 +180,40 @@ def update_section(framework_slug, service_id, section_id):
 
     posted_data = section.get_data(request.form)
 
-    try:
-        data_api_client.update_service(
-            service_id,
-            posted_data,
-            current_user.email_address)
-    except HTTPError as e:
-        errors = section.get_error_messages(e.message)
-        if not posted_data.get('serviceName', None):
-            posted_data['serviceName'] = service.get('serviceName', '')
+    errors = None
+    uploaded_documents, document_errors = upload_service_documents(
+        s3.S3(current_app.config['DM_DOCUMENTS_BUCKET']),
```

Comment:
It seems odd to me that the FE apps should have to know about S3. Could the `upload_service_documents` util not just pass in the `current_app.config` and the util function deal with whatever storage implementation we happen to have? Not suggesting we change that now, but maybe one for the tech debt pile...

Reply:
I agree wholeheartedly that this isn't for now :grinning: